acquia/acquia_cms
# ACMS-1082: Add API modules to ACMS.

**Motivation**
Adds some additional API modules and sets as dependencies of ACMS Headless.

**Proposed changes**
- Adds [OpenAPI UI](https://www.drupal.org/project/openapi_ui) module.
- Adds [ReDoc for OpenAPI UI](https://www.drupal.org/project/openapi_ui_redoc) module.
- Adds [OpenAPI JSON:API](https://www.drupal.org/project/openapi_jsonapi) module.
- Sets ReDoc for OpenAPI UI and OpenAPI JSON:API as composer dependencies both in ACMS and ACMS Headless.
- Sets as installed dependencies in ACMS Headless info yaml.
- 
**Alternatives considered**
N / A

**Testing steps**
- Pull down branch
- Install the ACMS Headless module
- Verify that ReDoc for OpenAPI UI, OpenAPI JSON:API, OpenAPI UI, and any other dependencies are installed along with ACMS Headless.

**Merge requirements**
